docs/user/rstudio/ide/guide/ui/files.qmd

```diff
@@ -20,26 +18,26 @@ RStudio supports syntax highlighting and other specialized code-editing features
 
 To create a new file you use the **File -\> New File** menu:
 
-![https://support.rstudio.com/hc/en-us/article_attachments/201460948/Screen_Shot_2015-04-02_at_3.55.43_PM.png](images/system-files-menu.png){alt=""}
+![](images/system-files-menu.png){fig-alt="A screenshot of the File Menu dropdown which has features like New File, Open File, Save File."}
```

Comment:
Since this is alt text it isn't a huge deal, but we should used "drop-down", not drop down or dropdown:
https://rstudiopbc.atlassian.net/wiki/spaces/ENG/pages/36113607/Documentation+Style+Guide#Terminology-references

Reply:
Changed